docs/avatars.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
```

Comment:
Why is avatar HTML in this branch? Name of branch is for Badge and Button
Suggest to create another branch and append Avatar changes there, as only max 2 components can be on one branch

Reply:
hey, I made all the components earlier in their individual branch. Right now I'm doing docs only for the components that were already made in their individual branches. 
But doing PR only 2 components at a time from the `docs` branch.

Reply:
Discussed and proposed suggesting to change PR title and description. 
Approved and can close this conversation 👍